workflows/reference_files/2.4/reference_files_header.smk

```diff
@@ -315,6 +315,23 @@ rule download_liftover_chains:
         gzip -dc > {output.chains}
         """)
 
+rule download_sdf: 
```

Comment:
good find getting them online. I thought we had to make our own sdf references!

Reply:
Supposedly the online ones have additional annotations that you can't get from making your own. I don't know how much benefit it would be for Starfish, but 🤷